ExampleMod/Content/Tiles/Furniture/ExampleChest.cs

```diff
@@ -41,6 +41,12 @@ public class ExampleChest : ModTile
 			AddMapEntry(new Color(200, 200, 200), this.GetLocalization("MapEntry0"), MapChestName);
 			AddMapEntry(new Color(0, 141, 63), this.GetLocalization("MapEntry1"), MapChestName);
 
+			// Style 1 is ExampleChest when locked. We want that tile style to drop the ExampleChest item as well. Use the Chest Lock item to lock this chest.
+			// No item places ExampleChest in the locked style, so the automatically determined item drop is unknown, this is why RegisterItemDrop is necessary in this situation. 
+			RegisterItemDrop(ModContent.ItemType<Items.Placeable.Furniture.ExampleChest>(), 1);
+			// Sometimes mods remove content, such as tile styles, or tiles accidentally get corrupted. We can, if desired, register a fallback item for any tile style that doesn't have an automatically determined item drop. This is done by passing in -1 as one of the tile styles or omitting the tileStyles parameter.
```

Comment:
> This is done by passing in -1 as one of the tile styles or omitting the tileStyles parameter.

Here and in the description, you should say 'omitting the style parameter' instead of '-1'